letsencrypt/acme/other.py

```diff
@@ -55,7 +100,7 @@ def from_msg(cls, msg, key, nonce=None):
         logging.debug('%s signed as %s', msg_with_nonce, sig)
 
         return cls(alg='RS256', sig=sig, nonce=nonce,
```

Comment:
Future note - 'RS256' - We will want to be thinking about how we can make this adapt to the key rather than relying on hardcoded parameters.  (I do see that there are already TODO items about RSA in JWK, but this will have to be adapted too)